### PR TITLE
[gala kinematic] Adding the end action

### PR DESCRIPTION
**Pull request by vincentpierre**
## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: habitat/core/batched_env.py [hunk 1]
```diff
@@ -399,7 +439,9 @@ def __init__(
         buffer_index = 0
 
         self.raw_rgb = None
+        self._raw_rgb = None
```
Comment (eundersander) [inline habitat/core/batched_env.py:442]: A bit confusing to have both `self.raw_rgb` and `self._raw_rgb`.

File: habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml [hunk 2]
```diff
@@ -3,23 +3,37 @@ CHECKPOINT_FOLDER: "/checkpoint/eundersander/gala_kinematic/ckpt/gala_kinematic_
...
 MAX_EPISODE_LENGTH: 500
+NUM_EPISODES: 10000
+MIN_NON_TARGET: 27
+MAX_NON_TARGET: 32
+FIXED_ROBOT_START_POSITION: False
+PREVENT_STOP_ACTION: False
 STAGGER: True
+TEST_EPISODE_COUNT: 10
+EVALUATION_MODE: False
+TRAIN_DATASET: ../data/train_generated.episode_set.json
```
Comment (eundersander) [inline habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml:23]: Will you add these data files to `gala_kinematic master`?

**Review by eundersander — APPROVED** [review]
I added a few comments but otherwise LGTM

File: habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml (hunk 2 above)
Comment (vincentpierre) [reply habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml:23]: I always override these when running code. We should get Andrew's dataset on master, then I will like the tidy house directly.

**Pull request merged**